stdlib/os/__init__.pyi

```diff
@@ -378,9 +377,7 @@ def fspath(path: str) -> str: ...
 def fspath(path: bytes) -> bytes: ...
 @overload
 def fspath(path: PathLike[AnyStr]) -> AnyStr: ...
-def get_exec_path(env: Mapping[str, str] | None = ...) -> List[str]: ...
-
-# NOTE: get_exec_path(): returns List[bytes] when env not None
+def get_exec_path(env: Mapping[str, str] | None = ...) -> list[str]: ...
```

Comment:
Was the comment wrong? Sounds like something we should handle with overloads.

Reply:
I tried `get_exec_path()`, `get_exec_path(None)`, `get_exec_path(os.environ)` and `get_exec_path(os.environb)`. They all returned a list of strings. I also don't think this is a windows thing, because windows generally uses byte strings than unixy operating systems.

Reply:
Thanks for checking! Maybe it was a Python 2 thing.

Reply:
That's what I thought too, but the function doesn't exist in Python 2. I also tried `git blame`, but that leads to 337abed05  which copies stubs from a different repo and gives no more history.